components/gc-follow-us/followus.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @title WET-BOEW Follow us component
+ * @overview Plugin used to replace Twitter with "X" - Deprecated
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @garneauma
+ */
+( function( $, window, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "followus",
+	selector = "." + componentName,
+	initEvent = "wb-init " + selector,
+
+	/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+
+		// Start initialization
+		// returns DOM object = proceed with init
+		// returns undefined = do not proceed with init (e.g., already initialized)
+		var elm = wb.init( event, componentName, selector ),
+			$elm;
+
+		if ( elm && event.currentTarget === event.target ) {
+			$elm = $( elm );
+
+			elm.querySelector( ".twitter .wb-inv" ).innerHTML = "X";
```

Comment:
Would be safer if we only replace the word "Twitter" by "X" instead of assuming the word "twitter" is there?

Something like:
```suggestion
			// Swapping the text "Twitter" for "X"
			var twitterElm = elm.querySelector( ".twitter" );
			twitterElm.innerHTML = twitterElm.innerHTML.replace( "Twitter", "X" );
```

Reply:
Agreed. I also added a condition to check if the twitter query returned something because otherwise it would throw an error.